pkg/eval/builtin_special.go

```diff
@@ -316,7 +316,12 @@ func (op useOp) exec(fm *Frame) Exception {
 	return nil
 }
 
+// TODO: Add support for module specs relative to a package/workspace.
+// See https://github.com/elves/elvish/issues/1400.
```

Comment:
This references a closed issue, which is confusing. I filed https://b.elv.sh/1421 that can be referenced here instead.

Reply:
done